README.rst

```diff
@@ -73,21 +73,25 @@ Clone the Repository
 
 To clone this repository you may use the following: ::
 
-    git clone git@github.com:xmos/lib_xcore_math
+    git clone --recurse-submodules git@github.com:xmos/lib_xcore_math
+
+.. note::
+
+   This repository uses submodules to include the XMOS CMake toolchain
 
 Including lib_xcore_math in External Applications
 -------------------------------------------------
 
-``lib_xcore_math`` can be compiled for both x86 platforms and xs3a architecture.
+``lib_xcore_math`` can be compiled for both x86 platforms and xs3a processors.
 
-On x86 platforms you can master DSP algorithms and test them for functional correctness;
+On x86 platforms you can develop DSP algorithms and test them for functional correctness;
 this is an optional step before porting the library to an xs3a device.
 
 .. note::
 
-   The instructions in this section refer to the compilation for xs3a; the steps to compile on the x86 platforms are the same,
+   The instructions in this section refer to the compilation for xs3a; the steps to compile for the x86 platforms are the same,
    except that the XTC build tools are not necessary and the default toolchain must be used. This can be done by skipping the option
-   ``-DCMAKE_TOOLCHAIN_FILE=../etc/xmos_cmake_toolchain/xs3a.cmake``  when comfiguring CMake.
+   ``--toolchain=../etc/xmos_cmake_toolchain/xs3a.cmake``  when comfiguring CMake.
```

Comment:
I'm unfamiliar with the `--toolchain` option.  Is it equivalent to defining the `CMAKE_TOOLCHAIN_FILE` symbol?

Reply:
yes, it is. I find `--toolchain` easier to remember.

Reply:
screenshot from the help menu
![image](https://github.com/xmos/lib_xcore_math/assets/93251216/ef093965-2eef-4f72-8aa2-6b6bf492f10a)

Reply:
Sounds good to me!